src/DynamicNLPModels.jl

```diff
@@ -205,39 +235,49 @@ function LQDynamicModel(dnlp::LQDynamicData{T,S,M}; condense = false) where {T,S
     Qf = dnlp.Qf
     ns = dnlp.ns
     nu = dnlp.nu
+    E  = dnlp.E
+    F  = dnlp.F
 
     sl = dnlp.sl
     su = dnlp.su
     ul = dnlp.ul
     uu = dnlp.uu
+    gl = dnlp.gl
+    gu = dnlp.gu
 
 
     if condense == false
-        H = _build_H(Q, R, N; Qf=Qf)
-        J = _build_J(A, B, N)
+        H  = _build_H(Q, R, N; Qf=Qf)
+        J  = _build_J(A, B, N)
```

Comment:
Instead of `J` and `EF`, let's call it `J1` and `J2` The current naming can potentially cause a confusion